api.go

```diff
@@ -99,19 +97,7 @@ type Uint256Quantity = uint256.Int
 
 type Data = hexutil.Bytes
 
-type PayloadID uint64
-
-func (id *PayloadID) UnmarshalJSON(text []byte) error {
-	return (*hexutil.Uint64)(id).UnmarshalJSON(text)
-}
-
-func (id *PayloadID) UnmarshalText(text []byte) error {
-	return (*hexutil.Uint64)(id).UnmarshalText(text)
-}
-
-func (id PayloadID) MarshalText() ([]byte, error) {
-	return hexutil.Uint64(id).MarshalText()
-}
+type PayloadID = hexutil.Bytes
```

Comment:
Strictly speaking this can be a fixed-length 8 byte ID. Checking the length would help catch some clients with wrong payload id lengths. Will update in later PR.